lib/ansible/config/base.yml

```diff
@@ -1745,6 +1757,15 @@ USE_PERSISTENT_CONNECTIONS:
   ini:
   - {key: use_persistent_connections, section: defaults}
   type: boolean
+VARIABLE_PLUGINS_ENABLED:
+  name: Vars plugin whitelist
+  default: ['host_group_vars']
+  description: Whitelist for variable plugins that require it.
+  env: [{name: ANSIBLE_VARS_ENABLED}]
+  ini:
+  - {key: vars_plugins_enabled, section: defaults}
+  type: list
+  version_added: "2.9"
```

Comment:
was this really added in 2.9 or did you mean 2.10 here?

Reply:
It was 2.9 at the time of opening the PR. I need to update these.